docker-compose.yml

```diff
@@ -0,0 +1,80 @@
+version: '3.7'
+networks:
+  dspacenet:
+services:
+  dspace:
+    container_name: dspace
+    image: "${DOCKER_OWNER:-dspace}/dspace:${DSPACE_VER:-dspace-7_x-jdk8-test}"
+    build:
+      context: .
+      dockerfile: Dockerfile.jdk8-test
+    depends_on:
+    - dspacedb
+    networks:
+      dspacenet:
+    ports:
+    - published: 8080
+      target: 8080
+    stdin_open: true
+    tty: true
+    volumes:
+    - assetstore:/dspace/assetstore
+    - ./dspace/src/main/docker-compose/local.cfg:/dspace/config/local.cfg
+  dspace-angular:
+    container_name: dspace-angular
+    depends_on:
+    - dspace
+    environment:
+      DSPACE_HOST: dspace-angular
+      DSPACE_NAMESPACE: /
+      DSPACE_PORT: '3000'
+      DSPACE_REST_HOST: dspace
+      DSPACE_REST_NAMESPACE: /
+      DSPACE_REST_PORT: '8080'
+      DSPACE_REST_SSL: "false"
+      DSPACE_SSL: "false"
+    image: dspace/dspace-angular:latest
+    networks:
+      dspacenet: {}
+    ports:
+    - published: 3000
+      target: 3000
+    - published: 9876
+      target: 9876
+    stdin_open: true
+    tty: true
+    volumes:
+    - ./dspace/src/main/docker-compose/environment.dev.js:/app/config/environment.dev.js
```

Comment:
Are we sure that we need to *both* override `environment.dev.js` AND set `DSPACE_REST_*` environment variables?  I would think that the `DSPACE_REST_*` environment variables would take precedent, which may mean this `environment.dev.js` file could go away?

Reply:
@tdonohue , if I remember correctly, the environment variables are not working.  They should probably be commented out until that issue is fixed.  See https://github.com/DSpace/dspace-angular/issues/403